### DISCUSSION TITLE
SD+ enable Encoder to switch Profiles

### DISCUSSION THREAD

**Discussion by Deadeye5589** [elgatosf/elgato#47] [Ideas]
As a user I find it limiting, that I can not use the encoder to change my profiles. 
Now all my profiles need an additional back button that will go to a main profile where I can select my profiles. 
In many cases I could use up to 8 sub profiles and 4 main profiles if I could use the encoder push buttons to select a different profile. This would increase the flexibility of the SD+
Please include this functionality in future releases.

**Comment by vadim-su** [comment]
It looks like a good idea 👍

**Comment by UweSchwo** [comment]
In my use cases I have dozens of buttons on a lower level (in folders or specialized profiles) that should fulfill some action and than change back to the main profile. All this buttons need a multi action construction to realize an profile change. 
It would be much easier if we had an option for every button "after action change profile to". Alternatively the "change back" could be made as an optional standard for folders.

**Comment by GeekyEggo** [comment]
Hey @Deadeye5589, this is a nice idea.

With our GitHub discussions intended for our SDK, we ask feedback intended for our software, e.g. Stream Deck, be submitted via our [Submit a Feature Request](https://help.elgato.com/hc/en-us/articles/360060806532--Submit-a-Feature-Request-) page. This ensures it reaches the intended team as quickly as possible; whilst we can't promise all feedback will be implemented, we do take your feedback into consideration when planning for future releases.
